apps/prairielearn/src/pages/instructorRequestCourse/instructorRequestCourse.ts

```diff
@@ -12,24 +11,35 @@ import * as Sentry from '@prairielearn/sentry';
 import * as opsbot from '../../lib/opsbot';
 import * as github from '../../lib/github';
 import { config } from '../../lib/config';
-import { CourseRequestSchema, IdSchema } from '../../lib/db-types';
+import { CourseRequestSchema, IdSchema, UserSchema } from '../../lib/db-types';
+import { RequestCourse } from './instructorRequestCourse.html';
 
 const router = express.Router();
 const sql = loadSqlEquiv(__filename);
 
+const CourseRequestRowSchema = z.object({
+  course_request: CourseRequestSchema,
+  approved_by_user: UserSchema.nullable(),
+});
+export type CourseRequestRow = z.infer<typeof CourseRequestRowSchema>;
```

Comment:
I'd like to copy over the PT convention of putting these "row" schema in the `.html.ts` file. I don't have a strong reason for doing this except that we need to pick a convention and we've already got that one, so let's stick with it.

Reply:
Thanks, I wasn't sure where to put it. I moved it to `.html.ts` to avoid circular dependencies.